SayMeYourName.java

```diff
@@ -0,0 +1,19 @@
+/**
+* output string name from command line 
+* Babak Alexey
+*/
+public class SayMeYourName {
+   //output string name from command line 
+   public static void main(String [] args) {
+      if(args.length == 0) {
+         System.out.println(" Please,enter your name ");

```

Comment:
That's not correct. You should print error message not ask user to enter name - application doesn't read it from console